crates/oxc_transformer/src/react_jsx/mod.rs

```diff
@@ -75,10 +77,16 @@ impl<'a> ReactJsx<'a> {
     ) -> Self {
         let imports = ast.new_vec();
         let options = options.with_comments(&ctx.semantic());
+
+        let import_source = options.import_source.clone().unwrap_or_else(|| "react".to_string());
+        let jsx_runtime_importer = format!("{import_source}/jsx-runtime");
```

Comment:
We can use Atom for this type, and then use `Atom::from_inline("react/jsx-runtime")` for the default case. This will remove all heap allocations, which saves a lot of performance.

Reply:
Can we use `Cow` instead?